exchanges/binance/binance_wrapper.go

```diff
@@ -387,11 +387,21 @@ func (b *Binance) FetchTradablePairs(ctx context.Context, a asset.Item) ([]strin
 		}
 		for u := range uInfo.Symbols {
 			if uInfo.Symbols[u].Status == "TRADING" {
-				curr, err := currency.NewPairFromString(uInfo.Symbols[u].Symbol)
-				if err != nil {
-					return nil, err
+				var curr currency.Pair
+				if strings.Contains(uInfo.Symbols[u].Symbol, currency.UnderscoreDelimiter) {
+					index := strings.Index(uInfo.Symbols[u].Symbol, currency.UnderscoreDelimiter)
```

Comment:
Might be able to strings.Split?

Reply:
By doing a split, we lose the delimiter. I'll go with your other option